test/integration/acceptance/query_permission_test_signatories.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "query_permission_test_signatories.hpp"
+
+QueryPermissionSignatories::QueryPermissionSignatories()
+    : QueryPermissionTestBase({Role::kGetMySignatories},
+                              {Role::kGetDomainSignatories},
+                              {Role::kGetAllSignatories}) {
+  for (int i = 0; i < 5; ++i) {
+    user_signatories_.emplace_back(
```

Comment:
With [std::generate_n](https://en.cppreference.com/w/cpp/algorithm/generate_n) it might be easier to read

Reply:
I am ok with emplace back, but 5 looks here as a magic constant :-/

Reply:
done